### DISCUSSION TITLE
Log level in env

### DISCUSSION THREAD

**Discussion by jjinkou2** [hiro5id/console-log-json#55] [Ideas]
Hello, 

would it be possible to set  a log level from an env var like this:

CONSOLE_LOG_LEVEL=info/warn/silly/...

**Comment by hiro5id** [comment]
Yeah I'll look into it.. works keeping me busy this week hope to check on it soon.

**Reply by hiro5id** [reply]
Also, I'm happy to take a PR if the feature is needed for you urgently.

**Comment by hiro5id** [comment]
You can now set the loglevel in the latest version. see the API Reference in the readme: https://github.com/hiro5id/console-log-json#api-reference
